docs/lib/auth/fragments/ios/signin_next_steps/20_confirm_sms_mfa.md

```diff
@@ -0,0 +1,41 @@
+If you get `confirmSignInWithSMSMFACode`, the next step is to verify the user through SMS multi-factor authentication. `AuthCodeDeliveryDetails` will contain information regarding where the code has been send. You should ask the user to provide the code that they received via SMS and invoke the `confirmSignIn` api with the SMS code:
+
+<amplify-block-switcher>
+
+<amplify-block name="Listener (iOS 11+)">
+
+```swift
+func confirmSignIn() {
+    Amplify.Auth.confirmSignIn(challengeResponse: "<confirmation code received via SMS>") { result in
+        switch result {
+        case .success(let signInResult):
+            print("Confirm sign in succeeded. Next step: \(signInResult.nextStep)")
```

Comment:
Here and throughout in other code samples:

```suggestion
            if signInResult.isSignedIn {
                print("Confirm sign in succeeded. The user is now signed in.")
            } else {
                print("Confirm sign in succeeded. Next step: \(signInResult.nextStep)")
                // Switch on the next step to take appropriate actions. If `signInResult.isSignedIn`
                // is true, the next step is 'done', and the user is now signed in.
            }
```

Please confirm that `done` and `isSignedIn` are the correct values.

Reply:
Updated